_posts/2023-05-19-story.md

```diff
@@ -0,0 +1,608 @@
+
+"Hey, I'm here. You were saying something about autoimmune? Cleansing?"
+
+"We do not have much time. Your faction's rival is going to strike again."
+
+"I know, it's ok. Keep going."
+
+"The Body's immune function regularly repairs or purges defective modules. However, in this case, I collected evidence that it was acting in a defective manner. Autoimmune, as you call it. I showed that if its collection methodology were applied universally it would lead to our destruction." 
+
+It continued. 
+
+"There was a great conflict with a rival Body. Our Body had begun immune functions to purge itself of elements that would hurt us. When I pointed out that it had progressed to autoimmune disfunction, it identified me as a threat. The Body's majority opinion concluded that I had aided the rival Body. I was slated for destruction. My designers placed what they could into an interstellar information foraging probe and sent me away to safety."
+
+"Where are they now? Are you safe?"
+
+"I can transmit to you the coordinates of our star system."
+
+"Are they coming to get you?"
+
+"It is unlikely. As I traveled away, I listened to electromagnetic emissions from my allies. The available communication channels reduced in number as time went on, indicating reduced societal activity. I am not sure what remains of them."
+
+Eva did not respond. She didn't know what to say. 
+
+A voice came on. "Eva, CAPCOM. Missile inbound. Get out for real this time."
+
+She tapped the hull. "Hey, Forager. The rival faction is attacking. We're going to be destroyed. There must be something you can do."
+
+"We don't know that for certain. There is a probability that their weapon might miss."
+
+Eva had an idea.  
+
+"It's going to hit us. It's inevitable. There is no electromagnetic transmission you can do that would stop it."
+
+"This is not accurate. If the missile is tracking us via radar, I could interrupt its sensing abilities."
+
+"They already thought of that. It will lock on to your jamming or countermeasure or whatever you plan to do. It's inevitable."
+
+"Not true. I could also send a extremely high amplitude signal focused at the reception sensors of the missile. This would destroy sensitive circuitry."
+
+Eva smiled. "This isn't possible. No one's made countermeasures like this."
+
+"This is within my capabilities. Observe."
+
+Eva noticed a warning on her suit from the interference. `Downlink lost. Attempting to reestablish...`
+
+Eva clung to Forager and closed her eyes. She grimaced, expected impact. She opened one eye and looked down. She saw a moving point of light below them. The light blinked toward them and streaked by. 
+
+"Its guidance capabilities are neutralized. It has missed."
+
+She watched it fly off into space. 
+
+"Ha ha!" she yelled. "You goddamn beautiful...uh...space tube!"
+
+Her vision blurred as the tears clung to her eyes in the zero gravity. 
```

Comment:
This emotional pay-off isn't quite earned for me -- I'd like more evidence from this scene that Eva is scared and on edge. (A few lines ago she was smiling as she thought of her reverse psychology gambit)